ERCS/erc-draft_fractional_nft.md

```diff
@@ -0,0 +1,348 @@
+---
```

Comment:
```suggestion
---
eip: 7651
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.